erpnext/patches.txt

```diff
@@ -630,7 +630,7 @@ execute:frappe.reload_doc('desk', 'doctype', 'dashboard')
 execute:frappe.reload_doc('desk', 'doctype', 'dashboard_chart_source')
 execute:frappe.reload_doc('desk', 'doctype', 'dashboard_chart')
 execute:frappe.reload_doc('desk', 'doctype', 'dashboard_chart_field')
-erpnext.patches.v12_0.add_default_dashboards
+erpnext.patches.v12_0.add_default_dashboards # 2020-05-05
```

Comment:
@nabinhait Will this rerun the patch during migrate?

Reply:
@scmmishra Yes it will rerun the patch